lora-codec-lansitec/readme.md

```diff
@@ -0,0 +1,514 @@
+﻿# Writing your own LPWAN Custom Codec using lpwan-custom-codec
+
+## Introduction
+
+Cumulocity IoT has the ability to integrate LPWAN devices via LPWAN agents. Supported list of LPWAN agents. (https://cumulocity.com/guides/protocol-integration/overview/)   
+
+In the following, we describe how to implement a custom codec microservice for implementing LPWAN device specific codec for decoding and encoding the device payload. 
+
+## Codec Workflow
+
+We have to deploy the codec microservice into Cumulocity where the codec microservice creates the device types with the list of supported device models.
```

Comment:
@BeateRixen What are the rules on "we" here? Do we continue to use "we recommend you to" or a direct addressing of the user?